content/en/docs/services/http-gateway/http-gateway.md

```diff
@@ -36,4 +36,7 @@ Here's an overview of the functionalities provided by the HTTP Gateway:
    - GetPendingCommands: Retrieves pending commands for devices.
    - CancelPendingCommands: Cancels resource commands.
 
+5. Service management:
+   - GetServicesMetadata: Retrieves metadata of service, which are running.
```

Comment:
typo:

"service, which is running" if we have a single service
"services, which are running" if we have multiple services